metadata/release-index.json

```diff
@@ -0,0 +1,17 @@
+{
+   "releases": [
+        {
+            "commit": "<hash>",
```

Comment:
From comment https://github.com/coreos/fedora-coreos-tracker/issues/98#issuecomment-497521551, I believe commit and version we are referring here is ostree commit and version. One thing to note that, we will have same ostree version number for a release across all arches but commit hash will be different for different arches.

Reply:
None of these fields are unique, we can have different commits with the same version and it won't conflict.